uaitrain/api/remove_train_job.py

```diff
@@ -15,37 +15,23 @@
 
 from uaitrain.api.base_op import BaseUAITrainAPIOp
 
-class RemoveUAITrainJobOp(BaseUAITrainAPIOp):
-    ACTION_NAME = "RemoveUAITrainJob"
-    """
-    RemoveUAITrainJobOp
-        Compatable with UAI Train RemoveUAITrainJob API func
-        Input:
-            pub_key             string(required) Public key of the user
-            priv_key            string(required) Private key of the user
-            project_id          int(optional)    Project ID of the job
-            region              string(optional) Which Region to run the job
-            zone                string(optional) Which Zone in the Region to run the job
-            job_id              string(required) Job id of the job
 
-        Output:
-            RetCode       int(required)                Op return code: 0: success, others: error code
-            Action        string(required)             Action name
-            Message       string(not required)         Message: error description
+class RemoveUAITrainJobApiOp(BaseUAITrainAPIOp):
+    """
+    RemoveUAITrainJobAPI
 
+        Identical with UAI Train RemoveUAITrainJob API func
+        Input:
+            TrainJobId              string(required)        Which train job to remove
     """
```

Comment:
Please describe return code for this OP. There must be cases job cannot be removed